packages/forms/src/UIForm/UIForm.component.test.js

```diff
@@ -123,22 +145,25 @@ describe('UIForm component', () => {
 
 		it('should set errors, applying widget errors hook', () => {
 			// given
-			const wrapper = shallow(<UIForm {...data} {...props} />);
+			const wrapper = mount(<UIForm {...data} {...props} />);
 			const newValue = 'toto is toto';
 			const event = { target: { value: newValue } };
 			const newErrors = { lastname: 'lol' };
 			props.onTrigger.mockReturnValueOnce(Promise.resolve({}));
 
 			// when
-			wrapper.instance().onFinish(
-				event,
-				{ schema: mergedSchema[0], value: newValue },
-				{
-					widgetChangeErrors() {
-						return newErrors;
+			wrapper
+				.find('TalendUIForm')
```

Comment:
Instead of mounting the whole component tree to find `TalendUIForm`, can you just shallow render `UIFormComponent` ?
Same for all following tests